xml/poc-requirements.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE chapter [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="poc-requirements">
+ <title>Skill requirements</title>
+<para>
+  Before you begin working through this guide, it is important to
+  understand your individual use case for your OpenStack deployment.
+  We recommend reading this guide with the following questions in mind:
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+        What problem am I trying to solve?
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        How does this solve my problem?
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        How do I want to implement my SUSE OpenStack Cloud?
+      </para>
+    </listitem>
+  </itemizedlist>
+  <para>
+    OpenStack is comprised of numerous projects that develop the services
+    that can make up your OpenStack cloud.
+  </para>
+ <informalfigure>
+  <mediaobject>
+   <imageobject>
+    <imagedata fileref="openstack-map.png"/>
```

Comment:
this doesnt render very well in pdf... looks fine in html

Reply:
Hmm that's what I was afraid of. I'm not too sure how to fix that - I'll get back to you.

Reply:
Fixed!